docs/worker-node/install-cvmfs.md

```diff
@@ -47,35 +47,43 @@ The following will install CVMFS from the OSG yum repository. It will also insta
         :::console
         root@host # yum install osg-oasis
 
-## Setup of FUSE and automount
+## Automount setup
 
-FUSE and automount are required for CVMFS and the steps to configure them are different on EL6 vs EL7. Follow the section that is appropriate for your host's OS:
+CVMFS uses automount, and the steps to configure it are different on EL6 vs EL7. Follow the section that is appropriate for your host's OS:
 
 * [For EL6 hosts](#for-el6-hosts)
 * [For EL7 hosts](#for-el7-hosts)
 
 ### For EL6 hosts
 
-1. Create or edit `/etc/fuse.conf` so that it contains:
+1. If automount is not yet in use on the system, do the following:
 
-        user_allow_other
+        :::console
+        root@host # chkconfig autofs on
+        root@host # service autofs start
 
-2. Create or edit `/etc/auto.master` to have the following contents to enable automounting:
+1. Create or edit `/etc/auto.master` to have the following contents to enable automounting cvmfs:
```

Comment:
Could we capitalize `CVMFS` here?

Reply:
Instead I removed the purpose from the instruction, there's no need for it.